### PR TITLE
feat: move subgraph

### PR DESCRIPTION
**Pull request by jordaniza**
This PR moves the tokenVoting subgraph files from OSx over.

In doing so it:

- Adapts the Schema
- Updates imports and directories
- Fixes a couple of tests
- Fetches the plugin addresss data from the commons configs

Task ID: [OS-482](https://aragonassociation.atlassian.net/browse/OS-482)

[OS-482]: https://aragonassociation.atlassian.net/browse/OS-482?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/subgraph/schema.graphql [hunk 1]
```diff
@@ -1,10 +1,327 @@
...
+  id: ID!
+  to: Bytes!
+  value: BigInt!
+  data: Bytes!
+  dao: Bytes!
+  proposal: IProposal!
+  execResult: Bytes
+}
+
+type StandardCallback @entity {
```
Comment (Rekard0) [inline packages/subgraph/schema.graphql]: do u need this?

File: packages/subgraph/manifest/data/mainnet.json [hunk 2]
```diff
@@ -8,6 +8,11 @@
         "address": "0xE978942c691e43f65c1B7c7F8f1dc8cDF061B13f",
         "startBlock": 16721812
       }
-    ]
+    ],
+    "Plugin": {
+      "name": "TokenVotingPlugin",
+      "address": null,
+      "startBlock": 16721812
```
Comment (Rekard0) [inline packages/subgraph/manifest/data/mainnet.json]: 2 question:
1- why we do not add the repo address of all supported network already here?
2- why we don't have "repo" wording in the names, isn't that confusing a bit

**Review by heueristik — DISMISSED** [review]
Let's first merge https://github.com/aragon/token-voting-plugin/pull/13

**Review by heueristik — COMMENTED** [review]
Let's first merge https://github.com/aragon/token-voting-plugin/pull/13

File: packages/subgraph/manifest/data/mainnet.json (hunk 2 above)
Comment (jordaniza) [reply packages/subgraph/manifest/data/mainnet.json]: Added it

File: packages/subgraph/src/plugin/plugin.ts [hunk 3]
```diff
@@ -1,24 +1,299 @@
...
+    proposalEntity.open = proposal.value.value0;
+    proposalEntity.executed = proposal.value.value1;
+
+    // ProposalParameters
+    const parameters = proposal.value.value2;
+    const votingMode = VOTING_MODES.get(parameters.votingMode);
+
+    // if (!votingMode) {
+    //   return;
+    // }
```
Comment (clauBv23) [inline packages/subgraph/src/plugin/plugin.ts]: could this comment be deleted?

File: packages/subgraph/src/plugin/plugin.ts [hunk 4]
```diff
@@ -1,24 +1,299 @@
...
+  event: VotingSettingsUpdated
+): void {
+  const packageEntity = TokenVotingPlugin.load(
+    generatePluginEntityId(event.address)
+  );
+  if (packageEntity) {
+    const votingMode = VOTING_MODES.get(event.params.votingMode);
+    // if (!votingMode) {
+    //   return;
+    // }
```
Comment (clauBv23) [inline packages/subgraph/src/plugin/plugin.ts]: same ^^

**Review by clauBv23 — APPROVED** [review]
LGTM!

**Review by heueristik — APPROVED** [review]

**Pull request merged**